cosmos/providers/dbt/parser/project.py

```diff
@@ -132,12 +132,12 @@ def __post_init__(self) -> None:
         for base_node in ast.find_all(jinja2.nodes.Call):
             if hasattr(base_node.node, "name"):
                 # check we have a ref - this indicates a dependency
-                if base_node.node.name == "ref" or base_node.node.name == "source":
```

Comment:
@tatiana @jlaneve  Ignore source because we not load source node from `yml` file. We can add it again after load source `at _handle_config_file`.

Reply:
@binhnq94 I think we actually need `source` for the open lineage support (8df9ee4) - I noticed another community member made a PR just with this change: #338. WDYT if we remove this change from this PR and have the discussion in that dedicated PR?

Reply:
Yeah, remove help avoid the logs here: https://github.com/astronomer/astronomer-cosmos/blob/main/cosmos/render.py#L241

Reply:
I will remove it.